<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeedLength</class>
 <widget class="QWidget" name="SeedLength">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>734</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,2,0,1,0,1,0,1,0,1">
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../resources_desktop.qrc">:/img/BigLogo@2x.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>8</number>
       </property>
       <item>
        <widget class="control::MwcLabelLarge" name="title">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Mnemonic Phrase Length</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="control::MwcLine" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>2</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>2</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>98</width>
         <height>88</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>221</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>221</width>
         <height>200</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="control::MwcRadioButton" name="radioLength24">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>170</y>
          <width>96</width>
          <height>20</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>24 Words</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcRadioButton" name="radioLength15">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>50</y>
          <width>96</width>
          <height>20</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>15 Words</string>
        </property>
       </widget>
       <widget class="control::MwcRadioButton" name="radioLength18">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>90</y>
          <width>96</width>
          <height>20</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>18 Words</string>
        </property>
       </widget>
       <widget class="control::MwcRadioButton" name="radioLength12">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>10</y>
          <width>96</width>
          <height>20</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>12 Words</string>
        </property>
       </widget>
       <widget class="control::MwcRadioButton" name="radioLength21">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>130</y>
          <width>96</width>
          <height>20</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="text">
         <string>21 Words</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>68</width>
         <height>189</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3,1">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="control::MwcLabelSmall" name="label_3">
       <property name="minimumSize">
        <size>
         <width>600</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot;line-height:138%;&quot;&gt;
&lt;p&gt;
Your mnemonic phrase is the key to accessing your wallet and funds. The number of words in your mnemonic phrase directly affects your wallet’s security:&lt;br /&gt;&lt;br /&gt;- &lt;span style=&quot; font-weight:600;&quot;&gt;12 words&lt;/span&gt;: Provides basic security. Suitable for most users but more vulnerable to advanced attacks.&lt;br /&gt;- &lt;span style=&quot; font-weight:600;&quot;&gt;18 words:&lt;/span&gt; Offers stronger security. A good balance between convenience and enhanced protection.&lt;br /&gt;- &lt;span style=&quot; font-weight:600;&quot;&gt;24 words:&lt;/span&gt; Provides maximum security. Recommended for protecting significant amounts or long-term holdings.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="control::MwcPushButtonNormal" name="continueButton">
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>40</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Continue</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelLarge</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcRadioButton</class>
   <extends>QRadioButton</extends>
   <header>control_desktop/MwcRadioButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLine</class>
   <extends>QFrame</extends>
   <header>control_desktop/MwcFrameWithBorder.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>radioLength12</tabstop>
  <tabstop>radioLength15</tabstop>
  <tabstop>radioLength18</tabstop>
  <tabstop>radioLength21</tabstop>
  <tabstop>radioLength24</tabstop>
  <tabstop>continueButton</tabstop>
 </tabstops>
 <resources>
  <include location="../resources_desktop.qrc"/>
 </resources>
 <connections/>
</ui>
